README.md

```diff
@@ -5,21 +5,32 @@
   <br />
 </p>
 
+<div align="center">
+
 [![Maven Central](https://img.shields.io/maven-central/v/com.amplitude/android-sdk)](https://mvnrepository.com/artifact/com.amplitude/android-sdk/latest)
+[![Legacy SDK](https://img.shields.io/badge/state-legacy-red)](https://github.com/amplitude/Amplitude-Kotlin)
```

Comment:
Can we put this to be the 1st?

Reply:
Until a majority of people are using the new SDK maybe we could color this yellow to be less scary? https://img.shields.io/badge/state-legacy-yellow